assets/src/edit-story/components/canvas/page.js

```diff
@@ -72,16 +85,67 @@ function Page() {
 			{ hasSelection && (
 				<Selection { ...selectionProps } />
 			) }
+			{ displayMoveable && (
+				<Moveable
+					target={ document.querySelector( '.selection' ) }
+					pinchThreshold={ 20 }
+					draggable={ true }
+					resizable={ true }
+					rotatable={ true }
+					onDrag={ ( { target, beforeTranslate } ) => {
+						frame.translate = beforeTranslate;
+						target.style.transform = `translate(${ beforeTranslate[ 0 ] }px, ${ beforeTranslate[ 1 ] }px)`;
+					} }
+					onDragStart={ ( { set } ) => {
+						set( frame.translate );
+					} }
+					onDragEnd={ () => {
+						setPropertiesOnSelectedElements( { x: selectionProps.x + frame.translate[ 0 ], y: selectionProps.y + frame.translate[ 1 ] } );
+					} }
+					onResizeStart={ ( { target, set, setOrigin, dragStart } ) => {
+						// @todo
+					} }
+					onResize={ ( { target, width, height, drag } ) => {
+						target.style.width = `${ width }px`;
+						target.style.height = `${ height }px`;
+						// @todo properly by setting frame, too.
+					} }
+					onResizeEnd={ () => {
+						// @todo Set the correct width/height.
+					} }
+					onRotateStart={ ( { set } ) => {
+						set( frame.rotate );
+					} }
+					onRotate={ ( { target, beforeRotate } ) => {
+						frame.rotate = beforeRotate;
+						target.style.transform = `rotate(${ beforeRotate }deg)`;
+					} }
+					onRotateEnd={ () => {
+						setPropertiesOnSelectedElements( { rotationAngle: frame.rotate } );
+					} }
+				/>
+			) }
 		</Background>
 	);
 }
 
 export default Page;
 
 function getUnionSelection( list, padding = 0 ) {
+	// Ignore multi-selection for now.
+	if ( 1 === list.length ) {
+		const { x, y, width, height, rotationAngle } = list[ 0 ];
+		return {
+			x: x - padding,
+			y: y - padding,
+			width: width + ( 2 * padding ),
+			height: height + ( 2 * padding ),
+			rotationAngle,
+		};
+	}
 	// return x,y,width,height that will encompass all elements in list
-	const { x1, y1, x2, y2 } = list
-		// first convert x1,y1 as upper left and x2,y2 as lower right
+	const { x1, y1, x2, y2, rotationAngle } = list
```

Comment:
IMHO we should just skip this entirely for now. The multi-select should be able to take care of multiple selection for us, we just need to render "stub" elements. I think I even noticed that Moveable draws outlines themselves so we may not even need a border.